common/templates/_deployment.yaml

```diff
@@ -22,7 +22,7 @@ spec:
         app.kubernetes.io/instance: {{ .Release.Name }}
     spec:
       imagePullSecrets:
-        - name: {{ .Values.ImagePullSecret}}
+        - name: {{ .Values.global.ImagePullSecret }}
```

Comment:
I'm not sure about this one.  It would not allow us to deploy ICONRelay in the same umbrella as iUS.  ICONRelay is published to a different registry with a different secret.